modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -25,155 +28,76 @@ Use the following example `PtpConfig` CR as the basis to configure `linuxptp` se
 
 . Create the following `PtpConfig` CR, and then save the YAML in the `boundary-clock-ptp-config.yaml` file.
 +
+[[ptp-boundary-clock]]
+.Recommended PTP boundary clock configuration
```

Comment:
Here and for `[[ptp-ordinary-clock]]` below: Reminder about [`[%collapsible]`](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#collapsible-content), if you're keen.